src/handlers.js

```diff
@@ -915,9 +1016,10 @@ const handlers = {
 	Literal(node, state) {
 		// TODO do we need to handle weird unicode characters somehow?
 		// str.replace(/\\u(\d{4})/g, (m, n) => String.fromCharCode(+n))
-		const value =
-			node.raw ??
-			(typeof node.value === 'string' ? JSON.stringify(node.value) : String(node.value));
+
+		let value = node.raw;
+		if (!value)
+			value = typeof node.value === 'string' ? JSON.stringify(node.value) : String(node.value);
```

Comment:
I don't understand what TS had a problem with the previous code had, but this works without type errors

Reply:
Doesn't seem to be a huge problem tho, resolving